vipps-recurring-api.md

```diff
@@ -392,6 +390,8 @@ See
 [Vipps landing page](https://github.com/vippsas/vipps-developers/blob/master/common-topics/vipps-landing-page.md)
 from Common topics, for more details about the landing page.
 
+**Note:** If payment should be required to activate an agreement, you need to specify an initial charge. If you are dealing with physical goods, this should be a RESERVE_CAPTURE, but for digital goods where the customer instantly gains access, DIRECT_CAPTURE might be easier to manage. See [Initial charge](#initial-charge)
```

Comment:
In other APIs we try to get away from "direct capture", as it's impossible to cancel then. See: https://vippsas.github.io/vipps-developer-docs/docs/vipps-developers/common-topics/reserve-and-capture